src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/AddressParsingTests.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace System.Net.NetworkInformation.Tests
 {
+    [SkipOnPlatform(TestPlatforms.Browser, "Not supported on Browser")]
```

Comment:
Does parsing addresses need something platform specific that isn't available on the browser?